### PR TITLE
notify_email field will now have email sent to that address for slow …

### PR DESCRIPTION
**Pull request by MattyKuzyk**
…rules and rule exceptions

### CODE REVIEW

**Comment by Qmando** [comment]
Can you add a call to  ``self.send_notification_email`` at line 642? (Exception when loading a new rule)

File: elastalert/elastalert.py [hunk 1]
```diff
@@ -1182,6 +1194,8 @@ def send_notification_email(self, text='', exception=None, rule=None, subject=No
         email = MIMEText(email_body)
         email['Subject'] = subject if subject else 'ElastAlert notification'
         email['To'] = ', '.join(self.notify_email)
+        if rule and rule['notify_email']:
```
Comment (Qmando) [inline elastalert/elastalert.py]: Can you check here that ``rule['notify_email']`` is not the same as ``self.notify_email``?

**Comment by Qmando** [comment]
I think the actual recipient addresses are on line 1204, ``smtp.sendmail(self.from_addr, self.notify_email, email.as_string())``, the rule's ``notify_email`` needs to be added there as well.

**Comment by MattyKuzyk** [comment]
Yeah, it would seem as though the fields on `email` I edited are just headers that don't affect the actual sending. Weird.

**Pull request merged**